### DISCUSSION TITLE
Price component

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#831] [Proposals]
En Mística no existe un componente dedicado a Precios.

Tendría sentido tener un componente de precio en el que cada marca pueda definir cuál es su estructura de precio?

La marca que no lo definan usarán un modelo estándar que propondremos desde Mística

Aquí están algunas exploraciones vistas con Canal Online
https://xd.adobe.com/view/ea696495-3fa2-4675-8d05-78ec0f9fe42b-bf3a/

**Comment by AnaMontes11** [comment]
<img width="830" alt="Captura de Pantalla 2022-10-07 a las 13 57 56" src="https://user-images.githubusercontent.com/88187691/194547951-ed12628d-91cf-494d-9351-a740301fc388.png">
Añado captura de lo que se propone desde marca para tenerlo en cuenta

**Reply by JaimeAinsa** [reply]
¿cuál es el modelo que se está proponiendo desde Mística? gracias

**Reply by yceballost** [reply]
Desde Mística aún no hemos trabajado esto en profundidad. Tuvimos una reu con vuestro equipo para tener una propuesta que estuviese alineada con la que habíais trabajado para Movistar. Nos faltaría hacer este ejercicio con el resto de marcas y ver si les encaja una propuesta global o tendríamos que trabajar una propuesta por marca.

Actualmente tenemos composiciones parecido a esto
![image](https://user-images.githubusercontent.com/6722153/195009816-33d35b62-b858-4810-90b5-1fc01a3c4ade.png)

**Comment by yceballost** [comment]
We are going to work on it
